templates/home/dashboard.html

```diff
@@ -30,6 +30,12 @@ <h3 class="text-center mb-3">Dashboard</h3>
         <div class="row justify-content-center mb-4">
             <div class="col-md-4">
                 <button id="continue_button" class="btn btn-primary btn-lg btn-block" type="submit">Continue</button>
+               <br>
+               <div class="row justify-content-center mb-2 ">
+            {% if subject_scheduled %}    
+            <p>Subject scheduled: <b>{{ subject_scheduled  }}</b></p>
+           {% endif %}
+                </div>
```

Comment:
Use proper indentaion.

Reply:
Also, you should create a separate row for this in Classwork Homework section below (see Classwork Homework comment).

Reply:
thank you, i made the required changes and made a completely new [pull request](https://github.com/garg3133/JagratiWebApp/pull/170), please review it